UseCase40/bin/setup-worker.php

```diff
@@ -13,7 +13,7 @@
 $finder->files()->in($discoverableDirectories);
 $jobCreator = (new Job\Type\Service())->addYmlServiceFinder($finder)->getNewJobTypeRegistrar();
 $jobCreator->setCode('protean_dlcp_example')
-    ->setWorkerClassUri(\Neighborhoods\KojoFitnessFunction42\V1\Worker\Facade::class)
+    ->setWorkerClassUri(\Neighborhoods\KojoFitnessUseCase40\V1\Worker\Facade::class)
```

Comment:
Odd diff, but everything is fine for `KojoFitnessUseCase40` and `KojoFitnessUseCase42`